process.c

```diff
@@ -3,60 +3,80 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <assert.h>
+#include <errno.h>
 
 #include "utils.h"
 #include "process.h"
 
 #define BUF_SIZE 100
 
-Process_t *create_process(char* pid) {
+int create_process(char *pid, Process_t **process) {
+  /*
```

Comment:
Again, be consistent. 
If you decided to add description with input parameters to the function - add it everywhere.